scene/gui/base_button.cpp

```diff
@@ -337,9 +335,6 @@ bool BaseButton::is_keep_pressed_outside() const {
 
 void BaseButton::set_shortcut(const Ref<ShortCut> &p_shortcut) {
 
-	if (shortcut.is_null() == p_shortcut.is_null())
```

Comment:
This test was completely bogus, as I understand it means that you could not set a new shortcut if both the existing one and the new one are valid (as `false == false` is `true`).